src/schema/v2/artwork/index.ts

```diff
@@ -809,6 +809,12 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           )
             return `Shipping: ${domesticShipping} ${shippingRegion} only`
 
+          if (
```

Comment:
```suggestion
          if (!artwork.domestic_shipping_fee_cents && !artwork.international_shipping_fee_cents) {
```
would cover undefined and null cases here

despite the suggestion what about the case where the `international_shipping_fee_cents` is undefined shouldn't we also cover it with || instead of &&?

Reply:
nevermind my suggestion, I see we have handling when the value is 0(false) this suggestion would mess with the current logic there